osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -272,12 +272,33 @@ public void SelectNext(int direction = 1, bool skipDifficulties = true)
             if (beatmapSets.All(s => s.Filtered.Value))
                 return;
 
+            if (selectedBeatmapSet == null || selectedBeatmap == null)
+            {
+                performInitialSelection(direction, skipDifficulties);
+                return;
+            }
+
             if (skipDifficulties)
                 selectNextSet(direction, true);
             else
                 selectNextDifficulty(direction);
         }
 
+        private void performInitialSelection(int direction, bool skipDifficulties)
```

Comment:
so this whole logic path is added but only ever executed in a test scene? that seems a bit suss to me. can we not just trigger selection on the root?